Source/IGListAdapter.m

```diff
@@ -308,6 +308,16 @@ - (NSUInteger)sectionForSectionController:(IGListSectionController <IGListSectio
     return [self.sectionMap sectionControllerForObject:object];
 }
 
+- (id)objectForSectionController:(IGListSectionController <IGListSectionType> *)sectionController {
+    IGAssertMainThread();
+    IGParameterAssert(sectionController != nil);
+    
+    NSUInteger section = [self.sectionMap sectionForSectionController:sectionController];
+    IGAssert(section != NSNotFound, @"Section nil for section controller %@", sectionController);
```

Comment:
@saraswatayu that method should have been updated ([source here](https://github.com/Instagram/IGListKit/blob/master/Source/IGListAdapter.m#L307-L311)) so it doesn't assert.

Are you looking at the `stable` branch?